daemon/src/position_metrics.rs

```diff
@@ -211,52 +210,44 @@ impl Cfd {
                 ..self
             },
             CetConfirmed => Self {
-                is_open: false,
-                is_closed: true,
+                state: AggregatedState::Closed,
                 ..self
             },
             RefundConfirmed => Self {
-                is_open: false,
-                is_refunded: true,
+                state: AggregatedState::Closed,
                 ..self
             },
             RevokeConfirmed => Self {
                 // the other party was punished, we are done here!
-                is_open: false,
-                is_closed: true,
+                state: AggregatedState::Closed,
                 ..self
             },
             CollaborativeSettlementConfirmed => Self {
-                is_open: false,
-                is_closed: true,
+                state: AggregatedState::Closed,
                 ..self
             },
             LockConfirmedAfterFinality => Self {
                 // This event is only appended if lock confirmation happens after we spent from lock
                 // on chain. This is for special case where collaborative settlement is triggered
                 // before lock is confirmed. In such a case the CFD is closed
-                is_open: false,
-                is_closed: true,
+                state: AggregatedState::Closed,
                 ..self
             },
             CetTimelockExpiredPriorOracleAttestation
             | CetTimelockExpiredPostOracleAttestation { .. } => Self {
-                is_open: false,
-                is_closed: true,
+                state: AggregatedState::Closed,
```

Comment:
🤔 Should we even set the `state` to "final" for anything other than `Confirmed` events, rejected and failed? Do we gain anything from reporting this stuff early in the metrics?

Reply:
We would see that a CFD is basically closed at the time the CET expired, not only when the closed transaction was confirmed.

Reply:
Is that preferable when looking at metrics? I just don't know.

Reply:
Yes, because we act on this information and would see discrepancies to our services built on top.